tests/test_tavern.py

```diff
@@ -3869,10 +3880,17 @@ def test_voljin_store_board(self):
         self.assertEqual(player_1.store[0].attack, player_1.in_play[0].base_attack)
         self.assertEqual(player_1.store[0].health, player_1.in_play[0].base_health)
 
+    class TestTickatusRandomizer(DefaultRandomizer):
+        def select_spell(self, spells: List[Type['Spell']]) -> Type['Spell']:
+            if GainArgentBraggart in spells:
```

Comment:
This doesn't seem right to me.  A randomizer should not be affecting any of it's inputs, it should just be returning one of them.